sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubIO.java

```diff
@@ -955,11 +961,14 @@ public void populateDisplayData(DisplayData.Builder builder) {
         super.populateDisplayData(builder);
 
         builder
-            .addIfNotNull(DisplayData.item("timestampLabel", timestampLabel))
-            .addIfNotNull(DisplayData.item("idLabel", idLabel));
+            .addIfNotNull(DisplayData.item("timestampLabel", timestampLabel)
+              .withLabel("Timestamp Label Attribute"))
+            .addIfNotNull(DisplayData.item("idLabel", idLabel)
```

Comment:
I've thought about this a bit; it has some pros/cons:

Pros:
* Less redundancy, less risk of versions getting out-of-sync
* Cleaner, clearer
* Easy to see a registry of all "common" display data items in one place.

Cons:
* Potential for accidental coupling. i.e., once you have a common display item for "Validation Enabled", it's easy to use that for everything. But it's harder to evolve when lots of things depend on it.
* Not entirely clear how to implement in order to get "registry" benefits, due to dynamic namespace.

Let's follow-up in another PR.